### PR TITLE
feat(llm): 📝 add the receive flow `NeedMemoTagModal`

### PR DESCRIPTION
**Pull request by thesan**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Add the "Need a Tag/Memo?" modal in the receive flow.

image
image

### ❓ Context

- [LIVE-9630]

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-9630]: https://ledgerhq.atlassian.net/browse/LIVE-9630?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by themooneer — APPROVED** [review]
👍

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  return (
+    <>
+      <Button type="accent" onPress={openModal}>
+        {t("transfer.receive.memoTag.link")}
+      </Button>
+
+      <BottomDrawer isOpen={isOpen} onClose={closeModal}>
+        <Flex alignItems="center" mb={7}>
+          <Flex
```
Comment (LucasWerey) [inline apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx:23]: Not sure but can't we use `Circle` from  "~/components/Circle";
 for this ?

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const openModal = () => setIsOpen(true);
+  const closeModal = () => setIsOpen(false);
+
+  return (
+    <>
+      <Button type="accent" onPress={openModal}>
+        {t("transfer.receive.memoTag.link")}
+      </Button>
+
+      <BottomDrawer isOpen={isOpen} onClose={closeModal}>
```
Comment (LucasWerey) [inline apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx:21]: We need to use QueuedDrawer instead

**Review by LucasWerey — CHANGES_REQUESTED** [review]

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx (hunk 1 above)
Comment (thesan) [reply apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx]: Yes that's better. I had only looked in the `native-ui`. Thanks

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx (hunk 2 above)
Comment (thesan) [reply apps/ledger-live-mobile/src/screens/ReceiveFunds/NeedMemoTagModal/NeedMemoTagModal.tsx]: Same here!

**Review by themooneer — APPROVED** [review]

**Review by LucasWerey — APPROVED** [review]

**Pull request merged**